test/passing/break_separators.ml

```diff
@@ -0,0 +1,119 @@
+[@@@ocamlformat "break-separators=after"]
+
+type t =
+  { aaaaaaaaaaaaaaaaaaaaaaaaa: aaaa aaaaaaaaaaaaaaaaaaa;
+    bbbbbbbbbbbbbbbbbbbbbbbbb: bbbbbbbbbbbb bbbb;
+    cccccccccccccccccccccc: ccccccc ccccccccccc cccccccc }
```

Comment:
And one more request, would it be possible to keep the `;` after the last element?

Reply:
@samoht that was my preferred style pre-ocamlformat, but it's involved to implement. If #406 was done, we'd be in a better position to do this sort of formatting in many instances. Note that "this sort" is referring to how the parsetree element for `{ ... }` involves negative indentation internally. More specifically, it is complicated because the format box nesting structure needed to produce the output you want does not match the nesting structure in the parsetree.

Reply:
To have a consistent style (and correct with respect to comments) there is no `;` printed after the last element of a record.